app/views/api/v1/topics/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Topics</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Image</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @topics.each do |topic| %>
```

Comment:
I don't think this view is needed.

Reply:
excuse me, the HTML files are not necessaries